tests/unit/test_calibration.py

```diff
@@ -226,11 +279,44 @@ def test_run_calibration() -> None:
         measurements=(test_data * scale) + offset,
         time=pl.Series(time_data).alias("time"),
     )
-    calibrator = calibration.Calibration(
+    calibrator = calibration.GgirCalibration(
         min_standard_deviation=9999, min_calibration_hours=1
     )
 
-    result = calibrator.run(dummy_measure)
+    result = calibrator.run_calibration(dummy_measure)
+
+    assert isinstance(
+        result, models.Measurement
+    ), f"was expecting type models.Measurement, object is of type {type(result)}"
+    assert np.allclose(
+        result.measurements, expected_data.measurements, atol=1e-3
+    ), "Measurement data did not match"
+    assert result.time.equals(expected_data.time), "Time series are not equal"
+
+
+def test_run_constrainedmincalibration() -> None:
+    """Testing run fcalibration fuinction for the constrained minimization."""
```

Comment:
couple typos

Reply:
So many typos in this whole PR, sorry. Multitasking got the best of me.